bundle/edu.gemini.programexport.servlet/build.sbt

```diff
@@ -0,0 +1,27 @@
+import OcsKeys._
+
+// note: inter-project dependencies are declared at the top, in projects.sbt
+
+name := "edu.gemini.programexport.servlet"
+
+// version set in ThisBuild
+
+unmanagedJars in Compile ++= Seq(
+  new File(baseDirectory.value, "../../lib/bundle/javax-servlet_2.10-2.5.0.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/osgi.cmpn-4.3.1.jar")
```

Comment:
And probably:

```scala
new File(baseDirectory.value, "../../lib/bundle/argonaut_2.11-6.2.jar")
```

Reply:
Given the amount of data we're extracting, I suspect you're right.